Libraries/DirectMLX.h

```diff
@@ -675,7 +683,7 @@ namespace dml
             }
 
             DML_GRAPH_DESC graphDesc = {};
-            graphDesc.InputCount = graph.inputCount;
+            graphDesc.InputCount = inputCount ? inputCount : graph.inputCount;
```

Comment:
What is the relationship between the parameter `inputCount` and `graph.inputCount`? In the nonzero case, should `inputCount` be `>=` than `graph.inputCount`? If so, should we assert that?

Reply:
Ah, yes, good point. The intent is that it should always be >= - I'll add an assert. This is useful in the case where you want (for whatever reason), additional unused inputs at the end.